File: algorithims/swap.js
var swap = function(array, firstIndex, secondIndex) {
    var firstChangedVal = array[firstIndex];
	array[firstIndex] = array[secondIndex];
	array[secondIndex] = firstChangedVal;
	return array;
};

var testArray = [7, 9, 4];
console.log( swap( testArray, 0, 1) );

File: algorithims/swap.rb
def swap ( array, first_index, second_index)
	first_swapped_val = array[first_index]
	array[first_index] = array[second_index]
	array[second_index] = first_swapped_val
	array
end

# testArray = [7, 9, 4]

#print swap( [7, 9, 4 ], 0, 1 )